documentation/decisions/adr/2023-09-07-data-replication-tool.md

```diff
@@ -1,14 +1,14 @@
 # Data Replication Strategy & Tool
 
-- **Status:** Active
-- **Last Modified:** 2023-09-07
+- **Status:** Draft
+- **Last Modified:** 2023-09-15
 - **Related Issue:** [#322](https://github.com/HHS/grants-equity/issues/322)
 - **Deciders:** Lucas Brown, Billy Daly, Sammy Steiner, Daphne Gold, Aaron Couch
 - **Tags:** Hosting, Infrastructure, Database
 
 ## Context and Problem Statement
 
-The Beta.Grants.Gov platform will need to consume data and apply additional load to the database that the database was not planned to support. Therefore, we will replicate the data for the Beta.Grants.Gov work so it will have a negligible impact on the database for replication related tasks and still provide up-to-date production data.
+The Beta.Grants.Gov platform will need to consume live grants data securely and without impacting grants.gov performance. However, the production database was not planned to support additional load to the database from the beta api. The beta work will also want to test schema changes to the database to facilitate new queries and lifecycle tracking that will not be possible in the production database. Additionally, the grants.gov database resides in another AWS account, which complicates access and security concerns.
 
 ## Decision Drivers
 
```

Comment:
Let's add a row about compliance with HHS security policies

Reply:
@lucasmbrown-usds and @acouch please let me know if the updated content addresses the hhs security compliance topic accurately.